chcemvediet/apps/inforequests/sitemaps.py

```diff
@@ -0,0 +1,90 @@
+from django.apps import apps as django_apps
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.contrib.sitemaps import Sitemap
+from django.db.models import Max
+from django.utils import translation
+
+from .models import Action, Inforequest
+
+
+class InforequestSitemap(Sitemap):
+    i18n = True
+    changefreq = u'weekly'
```

Comment:
Ak je inforequest `closed`, tak by sa uz nemal menit. Preto rozmyslam ze pre closed inforequesty by som nastavil attribut `changefreq` na `u'never'`.

Reply:
@viliambalaz Vsetky zverejnene infoziadosti budu closed a urcite sa bude obcas nieco menit aj v uzavretej zverejnenej infoziadosti. Takze `weekly` je tak akurat. Ak zistime, ze nas google prilis vytazuje, tak to zmenime z tyzdenne na mesacne.

Reply:
@viliambalaz Vsetky zverejnene infoziadosti budu closed a urcite sa bude obcas nieco menit aj v uzavretej zverejnenej infoziadosti. Takze `weekly` je tak akurat. Ak zistime, ze nas google prilis vytazuje, tak to zmenime z tyzdenne na mesacne.

Reply:
:ok: